Source/WebCore/html/HTMLPlugInElement.cpp

```diff
@@ -176,8 +176,7 @@ void HTMLPlugInElement::defaultEventHandler(Event& event)
     // Don't keep the widget alive over the defaultEventHandler call, since that can do things like navigate.
     {
         RefPtr<Widget> widget = downcast<RenderWidget>(*renderer).widget();
-        if (!widget)
-            return;
+        if (widget)
         widget->handleEvent(event);
```

Comment:
Please indent this line.

Reply:
Done - I also added platform specific expectation for Mac since for some reason, this test is failing on Mac but not other platforms. I looked into all "Test Expectations" file to see if this is getting skipped on other platforms but I couldn't find it.